pipeline.py

```diff
@@ -11,7 +11,18 @@
 import oneaday_filter
 import result_formatter
 import scraper_connection
-from petrarch import petrarch
+#from petrarch2 import petrarch2
+
+server_details, file_details, petrarch_version = utilities.parse_config('PHOX_config.ini')
```

Comment:
`petrarch` gets called in `main`, so keeping it inside `run` was making it complain. I also didn't want it to be called repeatedly, but I think putting it in `main` will be okay for that. Is that where you think it should go?